server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKSecurityTool.java

```diff
@@ -47,6 +49,7 @@
 class ZKSecurityTool {
   private static final Logger log = LoggerFactory.getLogger(ZKSecurityTool.class);
   private static final int SALT_LENGTH = 8;
+  private static final Charset CRYPT_CHARSET = Charset.forName("UTF-8");
```

Comment:
add a comment about why a new Charset instead of relying on `StandardCharsets.UTF_8`

Reply:
Switching to `StandardCharsets.UTF_8`